src/lib/Scenes/Home/index.tsx

```diff
@@ -133,13 +129,6 @@ export default class Home extends React.Component<Props, State> {
             <SalesRenderer render={renderWithLoadProgress(Sales)} />
           </Tab>
         </ScrollableTabView>
-
-        {showConsignmentsSash && (
-          <DarkNavigationButton
-            title="Sell works from your collection through Artsy"
-            onPress={this.openLink.bind(this)}
-          />
```

Comment:
Is it ok that we're removing the consignments sash completely?

Reply:
I feel like that's the original logic though, Stephen is gonna be away for a while, so not sure what to do with that though. That said, I think this was the only way you can start a consignment outside of an email - so might be worth waiting on someone with a definitive answer